test-suite/src/test/require-jdk8/org/scalajs/testsuite/javalib/time/chrono/IsoChronologyTest.scala

```diff
@@ -0,0 +1,106 @@
+package org.scalajs.testsuite.javalib.time.chrono
+
+import java.time.{Period, LocalDate, DateTimeException}
+import java.time.chrono.{IsoChronology, IsoEra}
+import java.time.temporal.ChronoField
+
+import org.scalajs.testsuite.javalib.time.temporal.TemporalTest
+
+object IsoChronologyTest extends TemporalTest {
+  final val iso = IsoChronology.INSTANCE
+
+  describe("java.time.chrono.IsoChronology") {
+    it("should respond to `getId`") {
+      expect(iso.getId).toEqual("ISO")
+    }
+
+    it("should respond to `getCalendarType`") {
+      expect(iso.getCalendarType).toEqual("iso8601")
+    }
+
+    it("should respond to `date`") {
+      for {
+        year <- Seq(Int.MinValue, -1000000000, -999999999, -1, 0,
+            1, 999999999, 1000000000, Int.MaxValue)
+        month <- Seq(Int.MinValue, 0, 1, 12, 13, Int.MaxValue)
+        day <- Seq(Int.MinValue, 0, 1, 28, 29, 30, 31, 32, Int.MaxValue)
+      } {
+        testTemporal(iso.date(IsoEra.CE, year, month, day))(LocalDate.of(year, month, day))
+        testTemporal(iso.date(IsoEra.BCE, 1 - year, month, day))(LocalDate.of(year, month, day))
+        testTemporal(iso.date(year, month, day))(LocalDate.of(year, month, day))
+        expectThrows[ClassCastException](iso.date(null, year, month, day))
+      }
+    }
+
+    it("should respond to `dateYearDay`") {
+      for {
+        year <- Seq(Int.MinValue, -1000000000, -999999999, -1, 0,
+          1, 999999999, 1000000000, Int.MaxValue)
+        day <- Seq(Int.MinValue, 0, 1, 365, 366, Int.MaxValue)
+      } {
+        testTemporal(iso.dateYearDay(IsoEra.CE, year, day))(LocalDate.ofYearDay(year, day))
+        testTemporal(iso.dateYearDay(IsoEra.BCE, 1 - year, day))(LocalDate.ofYearDay(year, day))
+        testTemporal(iso.dateYearDay(year, day))(LocalDate.ofYearDay(year, day))
+        expectThrows[ClassCastException](iso.dateYearDay(null, year, day))
+      }
+    }
+
+    it("should respond to `dateEpochDay`") {
+      for (day <- Seq(Long.MinValue, -365243219163L, -365243219162L, -1L, 0L,
+            1L, 365241780471L, 365241780472L, Long.MaxValue))
+        testTemporal(iso.dateEpochDay(day))(LocalDate.ofEpochDay(day))
```

Comment:
Is this because the generator expression runs over two lines? Otherwise, a one-line body should not be put in braces, right?